resources_portal/models/material_request.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+
+from .material import Material
+from .user import User
+
+
+class MaterialRequest(models.Model):
+    class Meta:
+        db_table = "material_requests"
+        get_latest_by = "created_at"
+
+    objects = models.Manager()
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    material = models.ForeignKey(
+        Material, blank=False, null=False, on_delete=models.CASCADE, related_name="requests"
+    )
+
+    requester = models.ForeignKey(
+        User, blank=False, null=False, on_delete=models.CASCADE, related_name="material_requests"
+    )
+
+    assigned_to = models.ForeignKey(
+        User, blank=False, null=False, on_delete=models.CASCADE, related_name="assigned_to_set"
+    )
+
+    mta = models.CharField(max_length=255, blank=True, null=True)
+    irb = models.CharField(max_length=255, blank=True, null=True)
+
+    # TODO: check type of this field
+    executed_mta = models.BooleanField(default=False)
```

Comment:
Hmm, good question. I forget what the purpose of this field was. Do you remember @dvenprasad ?

Reply:
The requester will need to sign the MTA once their request for the resource is approved. I think this is to track whether they had signed it or not.

Reply:
Hmm so does this summarize the flow?
  * An MTA (in the form of a PDF) is uploaded with the material.
  * A requester downloads that PDF, fills it out, and uploads it with their material request.
  * A reviewer approves the request.
  * The requester then reuploads the MTA but with it being signed this time.

If so, should we have the executed_mta field also be a PDF? This way instead of overwriting the original MTA that was uploaded with the request we just have a second one that is signed. One thing that concerns me is that someone could upload a "signed MTA" that isn't actually signed, we won't actually check it, but we'll interpret them uploading it as having signed it. Then when we notify the sharer that the requester uploaded the MTA we can say something like "make sure that the MTA was signed in all required locations" or something like that.

Reply:
No, it is
- Requester makes request, at this point they can download the MTA and read it. 
- A sharer/reviewer approves request
- Requester uploads the signed MTA to the portal (we can send them an email with the file, or they can download it on the portal) 
-  Sharer downloads signed MTA (with requester signatures) and also signs it.  

Yes, I agree we'll need to add that message asking both Requesters and Reviewers ensure that it signed in all places. We maybe need provide some mechanism to mark that they are missing signatures if the MTA isn't signed correctly.

Reply:
Ok so the `material_request.mta` field will be for the partially signed MTA that the Requester uploads, and the `material_request.executed_mta` field will be for the fully signed MTA that the Reviewer uploads?

Reply:
Can we name `material_request. mta`,  requester_signed_mta for clarity?

Reply:
Wait wouldn't that be `material_request.requester_signed_mta` and `material_request.reviewer_signed_mta` respectively?

Reply:
Yes, but `executed_mta` indicates that all parties have signed and it's ready to go vs `reviewer_signed_mta` which is implies reviewer has signed it and not clear that it is the final state of that pdf.

Reply:
ok, I renamed the field `mta` to `requester_signed_mta`. Will wait until we clarify how are we storing the pdfs (file field vs url+s3) before updating their types.